README.md

```diff
@@ -122,22 +138,28 @@ rvm install 2.7.6 --disable-binary --with-jemalloc
  Run the following command as the `postgres` user and then enter your houdini_user
  password at the prompt.
 
-> *Note*: For development, Houdini expects the password to be 'password'. This would be terrible
-for production but for development, it's likely not a huge issue.
+**Note: For development, Houdini expects the password to be 'password'. This would be terrible
+for production but for development, it's likely not a huge issue.**
 
-> *Tip*: To run this, add `sudo -u postgres ` to the beginning of the following command.
+#### Create user:
```

Comment:
This is creating a user for the postgres database. I worry that this header, as written, might be too confusing to people because they'll think they might think they've  added an Houdini user already.